server/src-lib/Hasura/RQL/Types/Permission.hs

```diff
@@ -36,7 +38,7 @@ instance Q.FromCol PermType where
     "delete" -> Just PTDelete
     _        -> Nothing
 
-permTypeToCode :: PermType -> Text
+permTypeToCode :: PermType -> T.Text
```

Comment:
nit: that's not required, we have an export of `Text` in the Prelude.

Reply:
Agreed, will fix this.